style.css

```
@@ -955,6 +955,34 @@ body.dark-mode .icon-sun {
+  background: transparent;
+  color: var(--accent);
+  font: inherit;
+  font-weight: 650;
+  text-decoration: underline;
+  text-decoration-color: color-mix(in srgb, var(--accent) 48%, transparent);
+  text-underline-offset: 0.18em;
+  cursor: pointer;
+}
+
+.detail-scientist-link:hover {
+  text-decoration-color: currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)